scripts/train/README.md

```diff
@@ -333,6 +344,27 @@ train_loader:
         ...
 ```
 
+To specify multiple streams (e.g. as in [Seamless data mixing](https://github.com/mosaicml/streaming?tab=readme-ov-file#seamless-data-mixing)) to load data in the yaml file, update your YAML like so:
+
+```yaml
+train_loader:
+    name: finetuning
```

Comment:
the finetuning dataloader doesn't actually support streams. not for any technical reason, just haven't update it, but this example should be in the pretraining data section and use the `text` dataloader

Reply:
@cli99 mind finishing this PR real quick?